cernopendata/modules/fixtures/data/docs/cms-releases-2016data-2024/cms-releases-2016data-2024.md

```diff
@@ -0,0 +1,52 @@
+The CMS experiment at CERN is proud to announce the first release of 13 TeV proton-proton
+collision data collected in 2016. Over 70 TB of 13 TeV [collision data](/search?q=&f=experiment%3ACMS&f=type%3ADataset%2Bsubtype%3ACollision&f=year%3A2016&l=list&order=desc&p=1&s=10&sort=mostrecent)
+and 830 TB of [corresponding simulations](/search?q=&f=experiment%3ACMS&f=type%3ADataset%2Bsubtype%3ASimulated&f=year%3A2016&l=list&order=desc&p=1&s=10&sort=mostrecent)
+are now accessible to the global scientific community and enthusiasts alike through the [CERN Open Data Portal](https://opendata.cern.ch/).
+
+<center>
```

Comment:
@jmhogan @katilp Thanks, I have amended the image to use the new SLUG, added "featured" so that the news item would appear first on the home page, edited the home page template to advertise "over five petabytes", and rebased and squashed.

There is one tentative content change that I did that I'd like to discuss with you. The leading illustrative image appeared right below the title, which did not look good on larger screens, because users did not see any text. I have done the following: (1) moved the first paragraph of the text to appear before the image, and (2) added image caption to explain what this candidate event is about (since it was having only photo credits, and casual visitors would not know about the content).

This is basically following what we have been doing in the past for the release announcements, e.g. see https://opendata.cern.ch/docs/cms-completes-run1-heavy-ion-open-data-collection

The new looks now give the following:

![13tev](https://github.com/cernopendata/opendata.cern.ch/assets/517546/30081e87-9ae0-4712-bf38-380a3d5bc084)

Are you OK with this change? Please let me know any amendments you'd like to bring to the image legend or whether you prefer I remove it and keep only photo credits, as in the original PR.

Reply:
Thanks! The change of order and the caption look good to me. Tom has an alternative image in https://cernbox.cern.ch/s/1rTQUFNLj8WQbpm but no need to change here if it is a trouble

Reply:
No problem, I can replace it here, please just let me know the desired image and the desired caption text.

Reply:
OK, @jmhogan @tpmccauley your preferences? Tom, do you have a caption for the new image?

Reply:
I have a slight preference for this image because it's more "full", but no real objection to changing either.

Reply:
OK, we can go with the present image and use the other one in other channels

Reply:
Thanks for the thoughts, merging "as is" then.